### DISCUSSION TITLE
RIP Copilot. Scam Student Developer Pack

### DISCUSSION THREAD

**Discussion by lauralex** [community/community#45322] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

# GitHub (Scam)Pilot stops working

I have the GitHub Student Developer Pack and suddenly GitHub Copilot stops working: now, it wants me to pay for it even though I've still got a valid Student license (wonderful).

This is the Scam page:
![image](https://user-images.githubusercontent.com/5181524/214543816-ef165719-1616-46b2-9c7d-cfef3399f594.png)

You can see that I still have a valid student license:
![image](https://user-images.githubusercontent.com/5181524/214544373-6eefa31b-5594-4442-8e60-7cc94e440fb0.png)

**Comment by jge162** [comment, marked as answer]
Sorry to hear and see this. U already submitted a ticket at the support page? Maybe they can fix it for you.

**Reply by lauralex** [reply]
> Sorry to hear and see this. U already submitted a ticket at the support page? Maybe they can fix it for you.

Yes, already done yesterday. thx

**Reply by jge162** [reply]
Np. I’m sure it will get fixed. Just the wait is frustrating I’m sure. Have a great rest of your day. Cheers.

**Reply by lauralex** [reply]
## Update
GitHub Copilot works inside the IDE, but it's like the account page is bugged for some reason. It still gives me the subscription page and I cannot change the settings related to GitHub Copilot.
![image](https://user-images.githubusercontent.com/5181524/214551925-eed5dc12-55e6-4280-a889-278419e79f2c.png)

**Reply by jge162** [reply]
That’s great news right. It works, albeit only partially   Progress! Hopefully once they reply your full access will be restored.

**Reply by Tatonta** [reply]
I'm having the same issues. Hopefully it'll get fixed soon, fingers crossed @lauralex
